modules/administration-guide/pages/uninstalling-che.adoc

```diff
@@ -21,20 +21,21 @@ To uninstall an instance of {prod} {prod-ver}:
 
 .Procedure
 
-. Obtain the name of the {prod-short} {orch-namespace} (default: `{prod-namespace}`):
+. Obtain the name of the {prod-short} {orch-namespace}. The default is `{prod-namespace}`.
 +
 [subs="+attributes,quotes"]
 ----
 $ {orch-cli} get checluster --all-namespaces \
   -o=jsonpath="{.items[*].metadata.namespace}"
 ----
 
-. Remove the {prod-short} instance from the __<{prod-namespace}>__  {orch-namespace}:
+. Remove the {prod-short} instance from its {orch-namespace}:
```

Comment:
I find the possessive form "its" confusing here. 

It leads the reader to think that the namespace belongs to the Che instance.

The namespace includes the Che instance.

Reply:
Thank you, Fabrice. Yes, you are right!